migrate.md

```diff
@@ -0,0 +1,111 @@
+- Ability to connect service instances to apps and service across different regions
+- Simplifies the capture of usage data per group
+
+If you created service instances before November 2018, then depending on the location in which your instance is hosted, it might be using Cloud Foundry instead of resource groups.
+
+## Migrating a service instance
+{: #migrate-task}
+
+If you want to learn more about the migration process before you begin, see the [IBM Cloud migration documentation ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-migrate){: new_window}.
+
+Only the person or group that created the instance or that has the appropriate privileges can migrate it. For more details, see [Required access for service instances](/docs/resources?topic=resources-migrate#required_access_instances){: new_window}.
+{: note}
+
+To migrate your service instance, complete these steps:
+
+1.  Determine which resource group you want to move the service instance to first.
+
+    See [Best practices for organizing resource in a resource group ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-bp_resourcegroups) for tips.
+
+    With a Premium plan, you can migrate multiple instances. However, the instances must all be migrated to the same resource group.
+
+1.  From the *Cloud Foundry Services* section of the {{site.data.keyword.Bluemix_notm}} resource list, click the migrate icon ![Migrate](images/migrate.svg) for the instance you want to migrate, and then click **Migrate** from the popup.
+
+1.  Click **Continue**, and then choose a resource group.
+
+    If you have not created a resource group, you can create one now. A **default** resource group is available. Take time to understand how the groups are used and create one if necessary. The resource group that you choose here *cannot* be changed later.
+
+1.  Click **Migrate**.
+
+    A message is displayed when the process is done. If you have other service instances to migrate, you can continue migrating other service instances, or click **Done**.
+
+1.  **Premium plan one-time step**: If the service instance you are migrating was created as part of a Premium plan, then you must inform the IBM support team that you are migrating a Premium plan instance. To do so, create a case from [IBM Cloud Support ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://cloud.ibm.com/unifiedsupport/supportcenter){: new_window}.
+
+    There are some additional steps that the IBM support team needs to take on your behalf. Add the following information to the case:
+
+    - Region in which the service instance is hosted, such as Dallas or Frankfurt
+    - Resource group name
+    - Resource group ID
+
+      If you don't know the ID, open the {{site.data.keyword.Bluemix_notm}} command line interface (CLI) tool and enter the following command:
```

Comment:
Consider changing the Bluemix conref here (and later in the file), too.  (I don't know what IBM style suggests in this case, but using a hyphen (dash?) in `command-line interface` is probably better for a compound adjective.)